test/jasmine/tests/gl2d_click_test.js

```diff
@@ -444,3 +444,149 @@ describe('Test hover and click interactions', function() {
         .then(done);
     });
 });
+
+describe('Test gl2d lasso/select:', function() {
```

Comment:
Painful! For some reason, these tests doesn't work on CI. Looks like the cases below can't even draw the scene:

![image](https://user-images.githubusercontent.com/6675409/27198066-b161b348-51de-11e7-999e-54ad2df69b16.png)

https://circleci.com/gh/plotly/plotly.js/4535#tests/containers/1

Can anyone try to pull down this branch and try this out locally?